pkg/function/create_volume_snapshot.go

```diff
@@ -134,7 +134,7 @@ func createVolumeSnapshot(ctx context.Context, tp param.TemplateParams, cli kube
 	}
 	wg.Wait()
 
-	err := fmt.Errorf(strings.Join(errstrings, "\n"))
+	err := fmt.Errorf("%s", strings.Join(errstrings, "\n"))
```

Comment:
It seems that this should use `errors.New`

```suggestion
	err := errors.New(strings.Join(errstrings, "\n"))
```

Question unrelated to this PR why are newlines (`\n`) being used inside the error message?

Reply:
Or maybe errkit, but I was trying to keep the changes minimal.

Reply:
Changed to errkit because we already have it in this file.